scipy/sparse/base.py

```diff
@@ -537,8 +557,32 @@ def __getattr__(self, attr):
         else:
             raise AttributeError(attr + " not found")
 
-    def transpose(self):
-        return self.tocsr().transpose()
+    def transpose(self, axes=None, copy=False):
+        """
+        Reverses the dimensions of the sparse matrix.
+
+        Parameters
+        ----------
+        axes : None, optional
+            This argument is in the signature *solely* for NumPy
+            compatibility reasons, as reversing the dimensions is
+            the only logical permutation that should be requested.
```

Comment:
As with `sum`'s `axis` parameter, it might be easier to just implement this. It's not particularly useful to have anything but `(1,0)`, but it matches the `np.matrix` behavior better.